service/controller/clusterapi/v29/resource/tcdp/template/params_main.go

```diff
@@ -1,6 +1,12 @@
 package template
 
-// ParamsMain is the data structure for the Control Plane Initializer template.
+// ParamsMain is the data structure for the Tenant Cluster Data Plane template.
```

Comment:
I'm still thinking about this naming... The `Data Plane` part somehow bothers me because it's a thing that exists in networking gear and doesn't really express concept of tenant cluster at least for me.
See:
* https://en.wikipedia.org/wiki/Forwarding_plane
* http://sdntutorials.com/difference-between-control-plane-and-data-plane/
* https://www.google.com/search?q=data+plane

Now that we are separating node pools from the tenant cluster control plane, I think it's fine to keep TCCP, but then call the other part just `Node Pool` i.e. `Node Pool Stack` or `Node Pool Template` as that's pretty self-describing.

Reply:
I created https://github.com/giantswarm/fmt/pull/60 as base for further discussions.